pkg/domain/infra/abi/containers.go

```diff
@@ -294,6 +284,27 @@ func (ic *ContainerEngine) ContainerRm(ctx context.Context, namesOrIds []string,
 		names = append(names, id)
 	}
 
+	// Attempt to remove named containers directly from storage, if container is defined in libpod
+	// this will fail, if options.Storage is not specified, code will fall through to removing
+	// the container from libpod.`
+	i := 0
```

Comment:
Please just make a new array - the memory savings/speed optimization here is going to be minor and these is a lot less readable than appending failed names to a fresh array.

Reply:
Fixed